### DISCUSSION TITLE
How to cross-language voice clone?

### DISCUSSION THREAD

**Discussion by bhimasta** [coqui-ai/TTS#2035] [General Q&A]
Hi, I am really new to voice cloning with very minimal knowledge about the code. 

As part of my academic research, I need to show a vignette at the beginning of my survey where I appear to speak a different language fluently (e.g., Chinese) in my original voice. 

Any insight or collab tutorial describing step by step on how to conduct these "cross-language voice cloning" is greatly appreciated.

Thank you very much.

**Comment by subbeddaemon** [comment]
I`m really interested too...

**Comment by titusfx** [comment]
@bhimasta @subbeddaemon any luck?

**Comment by evanbrociner** [comment]
Any updates on this?
